config.yaml

```diff
@@ -5,26 +5,37 @@ plugins: /verdaccio/plugins
 
 auth:
   openid-connect:
+    # Verdaccio public URL. If served on a subpath, make sure to include a
+    # trailing slash.
     publicUrl: http://localhost:4873
 
-    ###
-    # when set redis storage (for sessions and tokens) will be used.
-    # see also: fs_session_store_path, fs_token_store_path.
+    # Redis hostname
+    # When set redis storage (for sessions and tokens) will be used.
+    # see also: fsSessionStorePath, fsTokenStorePath.
     redisUri: redis
 
     ###
     # session files (persistent) storage path and token files (temporary for short-lived files) storage path: will be used if redisUri not set
     #fsSessionStorePath: /tmp
     #fsTokenStorePath: /tmp
 
+    # OpenID Connect Issuer URL
     issuer: https://keycloak/auth/realms/MyRealm/
+    # OpenID Connect Client ID.
+    # Alternatively can be set via environment variable OIDC_CLIENT_ID
     clientId: verdaccio
+    # OpenID Connect Client Secret.
+    # Alternatively can be set via environment variable OIDC_CLIENT_SECRET
     clientSecret: '...'
+    # OpenID Connect Scopes
     scope: 'openid profile email offline_access'
+    # Optional id_token claim that will be used for username
     usernameClaim: preferred_username
 
-    # optional roles claim (id_token).
-    # claim can be array or string (comma-separated)
+    # Optional switch to alternative login method, using access token as password.
+    # The username must be the same as the one used to acquire the access token,
+    # the password must be an OIDC access token. The e-mail address will be ignored.
+    # If not set, the regular authentication flow will be used.
     rolesClaim: roles
```

Comment:
The comment is not related to `rolesClaim` field

Reply:
FIXED